elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -280,12 +280,24 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                         continue
 
                     # Get corresponding property's value from parsed pipeline
-                    property_value = operation.component_params.get(component_property.ref)
+                    property_value_dict = operation.component_params.get(component_property.ref)
+                    property_value = property_value_dict[property_value_dict['activeControl']]
```

Comment:
Is this supposed to be:

```suggestion
                    property_value = property_value_dict['activeControl']
```

Reply:
Oh wait sorry, I mis-read this. But for some reason I'm still getting an error thrown in this scenario on submit

Reply:
are you testing using a completely new pipeline? Because this would probably break if your using a pipeline not created on this PR branch (just guessing without an error message)

Reply:
That's a good question, so I had to try again. Unfortunately, the same issue happened with a new pipeline, new component/different property under the same circumstances as above (entered a value, then removed it)

Reply:
@ajbozarth - I can confirm, after entering a value and deleting it, the resulting json payload sent to the backend is missing the default controller and value. 
```
'bash_command': {'activeControl': 'StringControl'},
```
it should be 
```
'bash_command': {'activeControl': 'StringControl',
                 'StringControl': ''
},
```

Reply:
I thought that was odd since it should behave the same as it does currently, so I tested out the same scenario in 3.2.2 and that how it behave there too. If you enter a value and then delete it the parameter will no longer be in the json (ie it's set to undefined). So in this case we should treat the key found in activeControl not being found the same as if the property key is not found. (OR we can discuss changing how we handle falsy values)